clipper_admin/clipper_admin/docker/docker_container_manager.py

```diff
@@ -335,12 +339,21 @@ def _add_replica(self, name, version, input_type, image):
 
         model_container_name = model_container_label + '-{}'.format(
             random.randint(0, 100000))
-        self.docker_client.containers.run(
-            image,
-            name=model_container_name,
-            environment=env_vars,
-            labels=labels,
-            **self.extra_container_kwargs)
+        if not self.gpu:
+            self.docker_client.containers.run(
+                image,
+                name=model_container_name,
+                environment=env_vars,
+                labels=labels,
+                **self.extra_container_kwargs)
+        else:
+            self.docker_client.containers.run(
+                image,
+                name=model_container_name,
+                environment=env_vars,
+                labels=labels,
+                runtime='nvidia',
```

Comment:
The only difference between two `run` function is this `runtime` arg. Why don't we add `runtime='nvidia` into `**extra_container_kwargs` or create a `runtime` variable that saves `nvidia` if `self.gpu` is on and some default value when `self.gpu` is False? In that way, we can avoid repetitive comment. For example, 

```
self.docker_client.containers.run(
                image,
                name=model_container_name,
                environment=env_vars,
                runtime=runtime,
                labels=labels,
                **self.extra_container_kwargs)
```

Reply:
Done